2019-12-16 01:29:26,891 INFO: Microblog startup [in /Users/bkhanal/microblog/app/__init__.py:48]
2019-12-16 01:29:28,543 ERROR: Exception on / [GET] [in /Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/app.py:1891]
Traceback (most recent call last):
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/Users/bkhanal/microblog/app/routes.py", line 37, in index
    prev_url=prev_url)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/bkhanal/microblog/app/templates/_post.html", line 31, in template
    {{ post.likes.count() }} likes
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/Users/bkhanal/microblog/app/templates/index.html", line 2, in top-level template code
    {% import 'bootstrap/wtf.html' as wtf %}
  File "/Users/bkhanal/microblog/app/templates/base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 1, in top-level template code
    {% block doc -%}
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 4, in block "doc"
    {%- block html %}
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 20, in block "html"
    {% block body -%}
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/flask_bootstrap/templates/bootstrap/base.html", line 23, in block "body"
    {% block content -%}
  File "/Users/bkhanal/microblog/app/templates/base.html", line 52, in block "content"
    {% block app_content %}{% endblock %}
  File "/Users/bkhanal/microblog/app/templates/index.html", line 15, in block "app_content"
    {% include '_post.html' %}
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/bkhanal/microblog/app/templates/_post.html", line 31, in template
    {{ post.likes.count() }} likes
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/parser.py", line 223, in parse_if
    drop_needle=True)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/parser.py", line 170, in parse_statements
    self.fail_eof(end_tokens)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/parser.py", line 104, in fail_eof
    return self._fail_ut_eof(None, stack, lineno)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/parser.py", line 90, in _fail_ut_eof
    self.fail(' '.join(message), lineno)
  File "/Users/bkhanal/microblog/venv/lib/python3.7/site-packages/jinja2/parser.py", line 59, in fail
    raise exc(msg, lineno, self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: Unexpected end of template. Jinja was looking for the following tags: 'endif'. The innermost block that needs to be closed is 'if'.
2019-12-16 01:30:01,324 INFO: Microblog startup [in /Users/bkhanal/microblog/app/__init__.py:48]
2019-12-16 01:54:47,006 INFO: Microblog startup [in /Users/bkhanal/microblog/app/__init__.py:48]
